### ISSUE TITLE
Ionic/Cordova - Windows UWP - Onedrive login/Windows Live - Invalid Redirect_Url / Window does not close after authentification,

### ISSUE THREAD

**Issue by MichaelPeter** [MrSwitch/hellojs-phonegap-demo#14]
Hello MrSwitch/ hello.js team :)

I am not shure if this is works as intended or if there is a solution for my request:
As redirect_url I specified a invalid url (which results in a 404). 

Result: In my UWP app after authentification the browser just gets white and nothing happens anymore. 

Is there a way to just close the authentification window or redirect the url to my own html inside the app, so I don't have to host a valid url? Can I access the Cordova InApp Browser Reference and do this with the corodva events?

ALSO:
I also tried the phonegap-demo with an cordova add windows platform (and added a self signed certificate to run if you want a sample I can provide it) and there after the windows authentifcation I was forwarded to the sample redirect page, but the authentification page was was never closed. The text just said you can close the window now. Is this intended or is there a better way?

tl;dr:
- Can I use the authentification without a valid web url? If not could a inapp self hosted localhost port work?

- The phonegap demo on windows UWP does not close the browser window. (I think also no log entries but will check again), is this intended or should the browser close after redirect?
